### ISSUE TITLE
Merge shapes

### ISSUE THREAD

**Issue by mararad** [jgraph/drawio#1697]
How to merge two shapes together? Most diagramming tools support merge, subtract shapes.

https://desk.draw.io/a/tickets/13057

**Comment by davidjgraph** [comment]
Closing through lack of additional interest.

**Comment by jmarinho** [comment]
To be honest, this is a feature I'd love to have in drawio.

**Comment by saftl** [comment]
I would love this feature as well

David Benson ***@***.***> schrieb am Fr. 26. Nov. 2021 um
16:34:

**Comment by IamPhytan** [comment]
+1 this will help for making complex shapes

**Comment by anugrahandi** [comment]
+1 for this feature

**Comment by dmiller-groq** [comment]
Yes, definitely a feature needed for a professional look to diagrams. I access drawio through my company portal, and for some reason couldn't access this issue from there, but I would say many of my co-workers want this. You have to consider that most users who want a feature won't comment here, they'll just find another tool with the features they want.

**Comment by pieveee** [comment]
Would love to have that too! +1

**Comment by OlaKats** [comment]
another +1 for this feature

**Comment by ChrisJipa** [comment]
count on more +1 for this feature

**Comment by stefanvogel65** [comment]
count on more +1 for this feature

**Comment by elgur** [comment]
Please kindly reconsider reopening this feature request, for the greater good. 
+1 for this feature

**Comment by davidjgraph** [comment]
Locking thread due to "+1" spam
